compiler/src/symboltable.rs

```diff
@@ -287,6 +361,7 @@ enum ExpressionContext {
     Load,
     Store,
     Delete,
+    NamedExprStore,
```

Comment:
Can't we use simply `Store` here? Why do we need an extra type?

Refer also here for the expression contexts: https://docs.python.org/3/library/ast.html?highlight=ast#module-ast

Reply:
Thats unnecessary=>cleaned up